config/locales/jobs/en.yml

```diff
@@ -2,7 +2,9 @@
 en:
   jobs:
     sms_otp_sender_job:
-      message: "%{code} is your %{app} one-time security code. This code will expire
+      create_account_message: "%{code} is your %{app} one-time security code to create an account. This code will expire
```

Comment:
"to create an account" is not future-proof once we add multiple phone numbers. How about "to confirm your phone number" or "to verify your phone number", or something along those lines.

Reply:
Great idea.  I'll add that.